Storage.md

```diff
@@ -0,0 +1,217 @@
+
+ItemsHoldr keeps track of items and their values while the game is running.
+It can also save this information to `localStorage`.
+Other websites also use `localStorage`, so ItemsHoldr has a prefix property to differentiate which information belongs to which.
+
+```typescript
+let ItemsHolder: ItemsHoldr = new ItemsHoldr({
+    prefix: "FullScreenShenanigans"
+});
+```
+
+## Items
+
+To add an item to the collection, use `addItem` and to remove an item, use `removeItem`.
+`setItem` is used to set the value of an item.
+
+```typescript
+ItemsHolder.addItem("color", { value: "red" });
+ItemsHolder.removeItem("color");
+ItemsHolder.setItem("color", "purple");
+```
+
+To manually update a changed item's value in localStorage, use `saveItem`.
+To save all the items' values in localStorage, use `saveAll`.
+
+```typescript
+ItemsHolder.saveItem("color");
+ItemsHolder.saveAll();
+```
+
+`increase` and `decrease` add and subtract respectively from the item's value.
+`toggle` flips the boolean value of the item.
+
+```typescript
+ItemsHolder.increase("weight", 14);
+ItemsHolder.decrease("weight", 10);
+ItemsHolder.toggle("started");
+```
+
+`checkExistence` can be used to check if there is an item under the specified key and if not, add it to the collection.
+
+```typescript
+ItemsHolder.checkExistence("color");
+```
+
+By default if an item does not exist when `setItem`, `getItem`, `increase`, or `toggle` are called, the item is then added to the collection.
+Define `allowNewItems` to explicitly enable or disable this feature.
+
+```typescript
+let ItemsHolder: ItemsHoldr = new ItemsHoldr({ allowNewItems: false });
+```
+
+## Updating
+
+Triggers are functions that get run when an item's value equals the specified value.
+
+```typescript
+ItemsHolder.addItem("color", {
+    value: "cyan",
+    triggers: {
+        red: () => { this.value = "black"; }
+    }
+});
+ItemsHolder.setItem("color", "red");
+```
+
+Modularity restricts an item's value to a range from 0 to the specified max and runs the `onModular` function as many times until the value hits the modular range.
+
+```typescript
+ItemsHolder.addItem("counter", { value: 0 });
+ItemsHolder.addItem("time", {
+    value: 0,
+    modularity: 12,
+    onModular: () => { ItemsHolder.increase("counter", 1); }
+});
+ItemsHolder.setItem("time", 100);
+```
+
+### Auto Save
+
+By default, the values in localStorage are not updated when their values change.
+Auto saving, which updates localStorage values, can be enabled when making the ItemsHolder container.
+
+```typescript
+let ItemsHolder = new ItemsHoldr({ autoSave: true });
+```
+
+To toggle the autoSave feature, use `toggleAutoSave`.
+
+```typescript
+ItemsHolder.toggleAutoSave();
+```
+
+## Clearing and Defaults
+
+To clear all items from the collection, use `clear`.
+
+```typescript
+ItemsHolder.clear();
+```
+
+To clear the collection, but keep some items which will always be in the collection, utilize ItemsHoldr's `valueDefault` property.
```

Comment:
> utilize ItemsHoldr's `valueDefault` property

I don't like that phrase in this context. Seems unnecessary. Switch to just `valueDefault`.